web/admin/src/Admin.js

```diff
@@ -199,6 +199,18 @@ export default class Admin extends PureComponent {
     else return "Add New Question"
   }
 
+  isButtonDisabled = () => {
```

Comment:
Now, remove the duplication between this function and the one above it :)

`returnHelpText = () => this.isButtonDisabled() ? 'Complete Last Question' : 'Add New Question'`

Then rename `isButtonDisabled` to something like `canAddNewQuestion`.  I'd have the `button` just be a different (disabled) button element when `canAddNewQuestion()` is false, with each having their proper text.